### ISSUE TITLE
Copy Assembler Registers does not work when game is not broke/paused

### ISSUE THREAD

**Issue by Symbai** [cheat-engine/cheat-engine#1129]
1. Set a breakpoint and let it hit
2. Remove breakpoint and continue/run
3. How do I copy an assembly register in the memory viewer on the right? It looks like there is currently no way and it only works when a breakpoint has been hit and the game is still broke/paused.

How about something very simple: Context menu with -> copy to clipboard -> copies the current text of the assembly register label to clipboard. 

This way CE doesn't even have to keep the registers in memory which I believe it doesn't do already and was the reason why we currently cannot open the "change register" dialog with a double click to copy the register.

**Comment by mgrinzPlayer** [comment]
Right mouse button on those labels should copy label text to clipboard:
```Pascal
procedure TMemoryBrowser.RegisterMouseDown(Sender: TObject;
  Button: TMouseButton; Shift: TShiftState; X, Y: Integer);
var s: string;
i: integer;
begin
  if button = mbright then
  begin
    if (sender is TLabel) then
    begin
      s:=trim(tlabel(sender).Caption);
      i:=pos(' ',s);
      if i>0 then //should always be true
      begin
        s:=copy(s,i+1,length(s));

        clipboard.AsText:=s;
      end;
    end;
  end;
end;
```

**Comment by Symbai** [comment]
So it silently copies the text? Nobody knows that without knowing he source code. This seems to be a bad idea. Especially since a context menu already exists.

I would suggest moving this into a new context menu entry and also make 

- Follow in disassembler
- Show in hexview

to work with label text too.

**Comment by mgrinzPlayer** [comment]
Yes. I think it would be better to move above procedure to new context menu entry, that way your clipboard won't be overwritten when you just want "Show in disassembler" or "Show in hexview" or change "Follow in hexview when stepping" option.

**Comment by BoomerangAide** [comment]
Any chance, despite it being from february 2020, that it get upgraded (to a legit context menu entry)?
In all cases, would it be possible to consider making a right-click (with our without menu, whatever is more affordable) on the "Registers" label copy **all** the registers?
Or an alternative: making the text selectable, so that it can be copied with a Ctrl+C (allow to get the register name along the way).

Recently, I was trying to debug something and tried to copy all stuff a called function would be passed as arguments. 
While copying ebp+x values (once I found how to display them) was not an issue (can just select a group and hit Ctrl+C), having to manually copy the registers one by one was harder.
I hope something will be done one day about this (even if I'm not sure how often I will get into that kind of use).
